src/App.tsx

```diff
@@ -46,54 +55,46 @@ const App: React.FC = () => {
     | FrameContexts
     | undefined;
 
-  const renderContent = (
-    isExtensionReady: boolean,
-    frameContext?: FrameContexts
-  ) => {
-    if (!isExtensionReady) {
-      return (
-        <Flex vAlign="center" hAlign="center" style={{ height: "100vh" }}>
-          <Loader label="Loading extension..." labelPosition="below" />
-        </Flex>
-      );
-    } else {
-      const { SidePanel, PreMeeting, Settings, Remove } = FrameContexts;
-      switch (frameContext) {
-        case SidePanel:
-          return <SidePanelPage />;
-        case PreMeeting:
-          return <PreMeetingPage />;
-        case Settings:
-          return <ConfigPage />;
-        case Remove:
-          return <RemovePage />;
-        default:
-          return null;
-      }
+  const renderContext = (frameContext?: FrameContexts) => {
+    const { SidePanel, PreMeeting, Settings, Remove } = FrameContexts;
+    switch (frameContext) {
+      case SidePanel:
+        return <SidePanelPage />;
+      case PreMeeting:
+        return <PreMeetingPage />;
+      case Settings:
+        return <ConfigPage />;
+      case Remove:
+        return <RemovePage />;
+      default:
+        return null;
     }
   };
 
+  const appStyles = {
+    width: "100%",
+    maxHeight: "100vh",
+    overflow: "auto",
+    overflowY: "auto",
+    overflowX: "hidden",
+    margin: "auto",
+    border: "none",
+    display: "flex",
+    flexDirection: "column",
+    backgroundColor: (msTeamsTheme.name === "dark" ? "rgb(32, 31, 31)" : undefined)
```

Comment:
**suggestion**: Let's add a comment here explaining why we need to hardcode this background color for the dark theme.